diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1db128ba..3301cf79 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -211,6 +211,7 @@ ragelmaker(src/parser/control_verbs.rl)
 configure_file(${CMAKE_MODULE_PATH}/config.h.in ${PROJECT_BINARY_DIR}/config.h)
 configure_file(src/hs_version.h.in ${PROJECT_BINARY_DIR}/hs_version.h)
 
+set(PKGCONFIG_EXTRA_LIBS "" CACHE STRING "Extra libraries to add to pkg-config Libs")
 configure_file(libhs.pc.in libhs.pc @ONLY) # only replace @ quoted vars
 install(FILES ${CMAKE_BINARY_DIR}/libhs.pc
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
diff --git a/chimera/libch.pc.in b/chimera/libch.pc.in
index 1eaf5bce..f3b37885 100644
--- a/chimera/libch.pc.in
+++ b/chimera/libch.pc.in
@@ -7,6 +7,6 @@ Name: libch
 Description: Intel(R) Chimera Library
 Version: @HS_VERSION@
 Requires.private: libhs
-Libs: -L${libdir} -lchimera
+Libs: -L${libdir} -lchimera @PKGCONFIG_EXTRA_LIBS@
 Libs.private: @PRIVATE_LIBS@
 Cflags: -I${includedir}/hs
diff --git a/libhs.pc.in b/libhs.pc.in
index 3ad2b90c..6d18b912 100644
--- a/libhs.pc.in
+++ b/libhs.pc.in
@@ -6,5 +6,5 @@ includedir=@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_INCLUDEDIR@
 Name: libhs
 Description: Intel(R) Hyperscan Library
 Version: @HS_VERSION@
-Libs: -L${libdir} -lhs
+Libs: -L${libdir} -lhs @PKGCONFIG_EXTRA_LIBS@
 Cflags: -I${includedir}/hs
